.github/workflows/windows.yml

```diff
@@ -0,0 +1,56 @@
+name: windows
+
+on:
+  push:
+    paths-ignore:
+    - '.github/workflows/**'
+    - '!.github/workflows/windows.yml'
+  pull_request:
+    paths-ignore:
+    - '.github/workflows/**'
+    - '!.github/workflows/windows.yml'
+
+jobs:
+  windows-msvc:
+    runs-on: windows-latest

```

Comment:
Change `windows-latest` to explicit windows version. We don't need sudden break in the ci due to version bump up which we don't track.